src/extension/extensions/CodeCompletionExtension.ts

```diff
@@ -21,6 +21,8 @@ import { getItemKind } from '../utilities/utilityFunctions';
 import { Logger } from '../utilities/winstonLogger';
 import { BaseExtension } from './BaseExtension';
 
+const REGEX_COMMENT = /^\s*(?:\/\/|\/\*\*|\*\/|\*)/;
```

Comment:
- Switching to non-capturing group for better perf (as we won't be needing the group capture)
- Stripping useless costly `g` flag